Translate #388 2016-07-08-weekly.md

devjin0617 opened the pull request:
처음작업해보는데 이렇게 진행하는게 맞는건가요?

devjin0617 committed:
Translate #388 2016-07-08-weekly.md

outsideris commented on weekly/_posts/2016-07-08-weekly.md:
```diff
@@ -0,0 +1,64 @@
+
+### Node v6.3.0 (최신버전) 릴리즈
+
+* [Node v6.3.0 (최신버전)](https://nodejs.org/en/blog/release/v6.3.0/). 이전 버전과 달라진 전체 변경사항은 [GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md)에서 확인할 수 있습니다.
```
6.3.0과 (최신버전) 사이에 공백을 제거해 주세요.(제목도요)

outsideris commented on weekly/_posts/2016-07-08-weekly.md:
```diff
@@ -0,0 +1,64 @@
+* [Node v6.3.0 (최신버전)](https://nodejs.org/en/blog/release/v6.3.0/). 이전 버전과 달라진 전체 변경사항은 [GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md)에서 확인할 수 있습니다.
+
+
+### Node.js 재단에서 Node.js Interactive Europe행사를 위한 keynote와 프로그래밍 발표
```
원문에도 프로그래밍이라고 되어 있기는 한데 영어에서는 그렇게 쓰는지 모르겠지만 내용상 보면 `프로그램`의 의미가 아닐까 합니다.

outsideris commented on weekly/_posts/2016-07-08-weekly.md:
```diff
@@ -0,0 +1,64 @@
+
+이벤트는 워크샵을 통해 공개행사를 진행할 것입니다. 커뮤니티, 기술대화, 그리고 Node.js, JavaScript 기능개발 사용방법들을 알릴 것입니다.
+
+[https://nodejs.org/en/blog/announcements/interactive-2016-ams/(https://nodejs.org/en/blog/announcements/interactive-2016-ams/) 여기에서 더 많은 정보를 확인할 수 있습니다.
```
링크에 대한 마크다운 문법이 잘못됐습니다. 링크의 경우에는 그냥 `<https://nodejs.org/en/blog/announcements/interactive-2016-ams/>`로 적어주셔도 됩니다. 

`여기에서`는 없어도 될것 같고 `<URL>에서`라고 적어도 될 것 같습니다.

outsideris commented on weekly/_posts/2016-07-08-weekly.md:
```diff
@@ -0,0 +1,64 @@
+[https://nodejs.org/en/download/](https://nodejs.org/en/download/) 에서 확인할 수 있습니다.
+
+### 커뮤니티 업데이트
+* [Node Hero - 디버깅 Node.js 애플리케이션](https://blog.risingstack.com/node-hero-node-js-debugging-tutorial/) : 당신은 시작하기를 통해 배울 수 있고, Node.js, deliver software products를 이용할 수 있습니다. Node.js에 관한 글을 쓰거나 발견했다면, [Evangelism 팀 저장소](https://github.com/nodejs/evangelism)의 [이슈페이지](https://github.com/nodejs/evangelism/issues)에 주간 업데이트 이슈로 알릴 수 있습니다.
```
동시에 진행되고 있어서 좀 애매하지만 같은 내용이라  https://github.com/nodejs/nodejs-ko/pull/391/files의 리뷰가 끝나면 내용을 맞춰주시면 좋을 것 같스빈다.

outsideris replied:
`Node.js에 관한 글을 쓰거나 발견했다면` 부분은 한줄 띄워서 다음 줄에 적어주세요.(공백라인 하나 있고 그다음 줄로 띄워야 된다는 의미입니다.)

devjin0617 committed:
Fix #388 2016-07-08-weekly.md

yous commented on weekly/_posts/2016-07-08-weekly.md:
```diff
@@ -0,0 +1,87 @@
+### Node.js Foundation Announces Keynotes and Programming for Node.js Interactive Europe
+-->
+
+### Node.js 재단에서 Node.js Interactive Europe행사를 위한 keynote와 프로그램 발표
```
Node.js Interactive Europe행사를 → Node.js Interactive Europe 행사를
keynote → 키노트

yous commented on weekly/_posts/2016-07-08-weekly.md:
```diff
@@ -0,0 +1,87 @@
+
+커뮤니티 다운로드는 기존 리틀 엔디언 바이너리에 빅 엔디언도 추가한 파워 시스템의 리눅스를 이제 포함합니다.
+
+<https://nodejs.org/en/download/> 에서 확인할 수 있습니다.
```
<https://...> 에서 → <https://...>에서

yous commented on weekly/_posts/2016-07-08-weekly.md:
```diff
@@ -0,0 +1,87 @@
+If you have spotted or written something about Node.js, do come over to our [Evangelism team repo](https://github.com/nodejs/evangelism) and suggest it on the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
+-->
+
+* [Node Hero - 디버깅 Node.js 애플리케이션](https://blog.risingstack.com/node-hero-node-js-debugging-tutorial/) : 당신은 시작하기를 통해 배울 수 있고, Node.js, deliver software products를 이용할 수 있습니다.
```
디버깅 Node.js 애플리케이션 → Node.js 애플리케이션 디버깅

yous replied:
':' 앞의 공백을 지워주세요.

원문은 '어떻게 Node.js를 시작하고, 이를 사용해 소프트웨어를 배포하는지 배울 수 있습니다'라는 뜻 같습니다.

yous commented on weekly/_posts/2016-07-08-weekly.md:
```diff
@@ -0,0 +1,87 @@
+
+* [Node Hero - 디버깅 Node.js 애플리케이션](https://blog.risingstack.com/node-hero-node-js-debugging-tutorial/) : 당신은 시작하기를 통해 배울 수 있고, Node.js, deliver software products를 이용할 수 있습니다.
+
+Node.js에 관한 글을 쓰거나 발견했다면, [Evangelism 팀 저장소](https://github.com/nodejs/evangelism)의 [이슈페이지](https://github.com/nodejs/evangelism/issues)에 주간 업데이트 이슈로 알릴 수 있습니다.
```
이슈페이지 → 이슈 페이지

yous commented on weekly/_posts/2016-07-08-weekly.md:
```diff
@@ -0,0 +1,87 @@
+### Upcoming Events
+-->
+
+### 이벤트
```
다가오는 이벤트

devjin0617 committed:
Fix #388 2016-07-08-weekly.md

- Node.js Interactive Europe행사를 → Node.js Interactive Europe 행사를
- keynote → 키노트

yous commented on weekly/_posts/2016-07-08-weekly.md:
```diff
@@ -0,0 +1,87 @@
+* [NodeConf Argentina](https://2016.nodeconf.com.ar): 18 - 19 November, 2016
+-->
+
+* [Node컨퍼런스 유럽(EU) 2016](http://www.nodeconf.eu/) : 10월 16일(일) - 20일(목) Lyrath Estate Kilkenny
```
'Node컨퍼런스 유럽(EU)'은 그냥 원문 그대로 'NodeConf EU'라고 적어도 될 것 같습니다.
':' 앞의 공백은 지워주세요.

10월 16일(일) - 20일(목) Lyrath Estate Kilkenny → 10월 16일(일)부터 20일(목)까지 아일랜드 Kilkenny의 Lyrath Estate에서 열립니다.

yous commented on weekly/_posts/2016-07-08-weekly.md:
```diff
@@ -0,0 +1,87 @@
+-->
+
+* [Node컨퍼런스 유럽(EU) 2016](http://www.nodeconf.eu/) : 10월 16일(일) - 20일(목) Lyrath Estate Kilkenny
+* [Node컨퍼런스 아르헨티나(Argentina)](https://2016.nodeconf.com.ar) : 2016년 11월 18-19일
```
Node컨퍼런스 아르헨티나(Argentina) → NodeConf 아르헨티나
':' 앞의 공백은 지워주세요.
2016년 11월 18-19일 → 2016년 11월 18일부터 19일까지 열립니다.

yous commented on weekly/_posts/2016-07-08-weekly.md:
```diff
@@ -0,0 +1,87 @@
+Have an event about Node.js coming up? You can put your events here through the [Evangelism team repo](https://github.com/nodejs/evangelism) and announce it in the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
+-->
+
+계획 중인 Node.js 있습니까? [Evangelism 팀 저장소](https://github.com/nodejs/evangelism)의 [이슈페이지](https://github.com/nodejs/evangelism/issues)에 주간 업데이트 이슈를 통해 알릴 수 있습니다.
```
계획 중인 Node.js 있습니까? → 계획 중인 Node.js 이벤트가 있나요?
이슈페이지 → 이슈 페이지

devjin0617 committed:
Fix #388 2016-07-08-weekly.md

- <https://...> 에서 → <https://...>에서
- 디버깅 Node.js 애플리케이션 → Node.js 애플리케이션 디버깅
- ':' 앞의 공백제거
- Node Hero - 디버깅 Node.js 애플리케이션 내용 수정
- 이슈페이지 → 이슈 페이지
- 이벤트 → 다가오는 이벤트

devjin0617 committed:
Fix #388 2016-07-08-weekly.md

- 'Node컨퍼런스 유럽(EU)' → 'NodeConf EU'
- 10월 16일(일) - 20일(목) Lyrath Estate Kilkenny → 10월 16일(일)부터 20일(목)까지 아일랜드 Kilkenny의 Lyrath Estate에서 열립니다.
- Node컨퍼런스 아르헨티나(Argentina) → NodeConf 아르헨티나
- 2016년 11월 18-19일 → 2016년 11월 18일부터 19일까지 열립니다.
- 계획 중인 Node.js 있습니까? → 계획 중인 Node.js 이벤트가 있나요?

devjin0617 committed:
Fix #388 2016-07-08-weekly.md

- 릴리즈 → 릴리스

devjin0617 committed:
Fix #388 2016-07-08-weekly.md

- 번역수정
